### DISCUSSION TITLE
GPU Requirements

### DISCUSSION THREAD

**Discussion by ravioliasb** [Project-MONAI/MONAI#7920] [Q&A]
I am new to Monai. I will be working with Monai and I am planning to build a PC. Would a RTX 4080 16gb be sufficient for all the Monai models, including the 3D segmentation models? 

Thank you.

**Comment by KumoLiu** [comment]
Welcome to MONAI! It will more depend on your task, for the models included in the model zoo, each model has a relevant part that demonstrates the requirements which you can refer to.
https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation#tensorrt-speedup

Thanks!
